lib/scan/dir/engine.py

```diff
@@ -179,6 +184,12 @@ def start(target, users, passwds, ports, timeout_sec, thread_number, num, total,
         trying = 0
         if target_type(target) != "HTTP":
             target = 'http://' + target
+        nowh3r3 = target + "/ThisFileIs404"
+        try:
+            r = SESSION.get(nowh3r3, verify=False, headers=user_agent, timeout=10)
```

Comment:
hardcoding 10 second timeout not right and it is too long - please use **timeout_sec** 
if user needs to increase the timeout they can always do that using command line options